### ISSUE TITLE
[Beats] Most beats/modules don't consistently use the "related" ECS object

### ISSUE THREAD

**Issue by BenB196** [elastic/beats#20616]
I was looking at the Okta Filebeat module that was introduced in 7.7.0 and noticed that it was using the [related.user & related.ip](https://www.elastic.co/guide/en/ecs/current/ecs-related.html) fields. However, after looking at other Filebeat modules and other beats (ex: Winlogbeat) it appears that the use of the related schema is inconsistent. Resulting in being unable to easily search across multiple data sources/modules.

From the definition of the [related](https://www.elastic.co/guide/en/ecs/current/ecs-related.html) in ECS, I would expect all modules/beats to take advantage of this. If not possible, I would expect modules/beats to be consistent and use the other provided ECS fields, in order to keep searching consistent across data sources/modules

**Comment by legoguy1000** [comment]
Do you have specific modules that aren't using the related fields??

**Comment by BenB196** [comment]
I just re-reviewed my notes since this issue was opened a while ago, and it appears that the related.user things that I had noted were fixed, but there are still a few issues with related.ip.

***Note***: I'm still running 7.10.0 beats currently, so I'm not 100% sure if the below issues were fixed in 7.10.x or 7.11.x

Auditbeat:
- Module: auditd

Event contains a source.ip value, but no related.ip value
```
    "auditd": {
      "summary": {
        "object": {
          "type": "user-session",
          "primary": "1.2.3.4", <------ IP #1 (source.ip)
          "secondary": "49383"
        },
        "how": "/usr/sbin/sshd",
        "actor": {
          "primary": "unset",
          "secondary": "root"
        }
      },
      "data": {
        "direction": "from-server",
        "ksize": "256",
        "lport": "22",
        "spid": "14453",
        "laddr": "5.6.7.8", <------ IP #2 (This is the host IP, but also part of the event, not sure if it should be included in the related.ip field)
        "pfs": "diffie-hellman-group-exchange-sha256",
        "rport": "49383",
        "op": "start",
        "mac": "hmac-sha1",
        "cipher": "aes256-ctr"
      },
      "message_type": "crypto_session",
      "sequence": 142962033,
      "result": "success"
    },
    ...
   "source": {
      "ip": "1.2.3.4",
    }
```

Winlogbeat:
- Module: security
  -  This module appears to be the greatest offender, in 24 hours I have 8,317,031 security events, all of which contain at least 1 IP in an ECS field, but no relate.ip field.

Event contains a source.ip value, but no related.ip value
```
    "event": {
      "created": "2021-03-18T13:22:52.794Z",
      "kind": "event",
      "provider": "Microsoft-Windows-Security-Auditing",
      "action": "kerberos-service-ticket-requested",
      "code": 4769,
      "module": "security",
      "category": "authentication",
      "type": "start",
      "outcome": "success"
    },
    "source": {
      "ip": "1.2.3.4" <------- IP is pulled from the message field of the event
    }
```
